src/components/add-transaction-dialog-select/add-transaction-dialog-select.html

```diff
@@ -0,0 +1,18 @@
+<div class="mdc-select" role="listbox">
```

Comment:
where is `add-transaction-dialog-select` class denoting that this is a component?